src/main/java/com/google/neighborgood/servlets/EditTaskServlet.java

```diff
@@ -66,12 +47,66 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
-    // Set the details and rewards to the newly input value
-    task.setProperty("detail", taskDetail);
-    task.setProperty("reward", rewardPts);
-    datastore.put(task);
+    // Edits tasks that have been claimed by setting the "helper" property to the userId
+    // of the helper and changing the task's status to "IN PROGRESS"
+    if (request.getParameter("action").equals("helpout")) {
+
+      UserService userService = UserServiceFactory.getUserService();
+      if (!userService.isUserLoggedIn()) {
+        System.err.println("User must be logged in to help out with a task");
+        response.sendError(
+            HttpServletResponse.SC_UNAUTHORIZED, "You must be logged in to help out with a task");
+      }
```

Comment:
I would suggest putting this block of code outside of the if-else statement because I added the login status check in a new commit, so at the end both if block and else block will need to check whether the user has logged into an account. As a result, check whether the user has logged into an account can be put at the very beginning of the `doPost()` method.

Reply:
Sounds good, I'll move this to outside of the if-else statement.